cmd/otelcol/main.go

```diff
@@ -56,8 +59,7 @@ func main() {
 	// TODO: Use same format as the collector
 	log.SetFlags(log.Ldate | log.Ltime | log.Lshortfile)
 
-	args := os.Args[1:]
-	if !contains(args, "-h") && !contains(args, "--help") {
+	if !contains(os.Args[1:], "-h") && !contains(os.Args[1:], "--help") {
```

Comment:
nit, why is this beneficial?

Reply:
What was the answer to this?

Reply:
Though the re-slicing is slightly inefficient, I prefer that to creating new variables. Plus, this code runs on starting only.